ui/packages/catalog-ui-search/src/main/webapp/component/search-form/forms-sharing/search-form-sharing-tab-container.view.js

```diff
@@ -15,10 +15,11 @@
 /*global require*/
 const Marionette = require('marionette')
 const template = require('../search-form.collection.hbs')
-const SearchFormCollectionView = require('./search-form-sharing.collection.view')
-const SearchFormsSharingCollection = require('./search-form-sharing-collection-instance')
+const SearchFormCollectionView = require('../search-form.collection.view')
```

Comment:
I wonder if we can reuse a single component instead of having `search-form-sharing-tab-container`, `search-form-system-tab-container`, and so on.  It looks like the only difference between them is their filters.

Reply:
I agree they are very similar but the system tab also has the `These are system search forms and cannot be changed` text.